packages/nextjs/src/config/webpack.ts

```diff
@@ -675,12 +661,14 @@ function addValueInjectionLoader(
   newConfig: WebpackConfigObjectWithModuleRules,
   userNextConfig: NextConfigObject,
   userSentryOptions: UserSentryOptions,
+  buildContext: BuildContext,
 ): void {
   const assetPrefix = userNextConfig.assetPrefix || userNextConfig.basePath || '';
 
   const isomorphicValues = {
     // `rewritesTunnel` set by the user in Next.js config
     __sentryRewritesTunnelPath__: userSentryOptions.tunnelRoute,
+    SENTRY_RELEASE: { id: getSentryRelease(buildContext.buildId) },
```

Comment:
l: can we move the `The webpack plugin's release injection breaks the `app` directory` comment down here?